src/app/ConcreteAttributePath.h

```diff
@@ -28,6 +28,11 @@ namespace app {
  */
 struct ConcreteAttributePath
 {
+    static constexpr EndpointId kInvalidEndpointId   = 0xFFFF;
```

Comment:
We now have this in multiple places.  Should we just move it into the header that declares the EndpointId type?

But also, do we actually need this, and the change the the default constructor?  There is no need I see for either one, or for anyone to use these values, offhand....

Reply:
Yes, moved to src/lib/DataModelTypes.h, also reuses  other values one from ClusterInfo.